.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ orbs:
 executors:
   with-chrome-and-firefox:
     docker:
-      - image: "cypress/browsers:node16.18.0-chrome107-ff106-edge"
+      - image: "cypress/browsers:node18.12.0-chrome103-ff107"
```

Comment:
the newer cypress orb should use less strict browser versioning, but that's a bigger update